### PR TITLE
Add pagination documentation

### LINKED ISSUE #27: Add documentation about pagination
Affected endpoints:

 - /v0/reserve/transactions
 - /v0/reserve/ledger
 - /v0/me/transactions
 - /v0/me/cards/:card/transactions

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
Closes #27.

### CODE REVIEW

File: _pagination.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Pagination
+
+All collection endpoints support [Range Pagination Headers](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) usgin `Range` & `Content-Range` entity-headers.
```
Comment (fixe) [inline _pagination.md]: Typo.

File: _pagination.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+List of endpoints that support pagination:
```
Comment (fixe) [inline _pagination.md]: I wouldn't add this here since it requires maintenance.

File: _pagination.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Pagination
+
+All collection endpoints support [Range Pagination Headers](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) using `Range` & `Content-Range` entity-headers.
+
+## Request
+
+You can provide the `Range` header specifying the items you want to retrieve.
```
Comment (fixe) [inline _pagination.md]: specifying how many items you want to retrieve

**Comment by fixe** [comment]
Thanks!

**Comment by nunorafaelrocha** [comment]
:wink:

**Comment by ruimarinho** [comment]
:+1:

**Pull request merged**